CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.2.0
+  - Support boolean to integer conversion #107
+  - Don't convert empty/null values to zero for integer conversions #68
```

Comment:
Should we leave it an unconverted string, or should we convert an empty string to nil? I think the latter would produce less surprising results, especially considering the following scenario:

 - field "foo" on event A is populated with a string representing an integer
 - field "foo" on event B is populated with an empty string
 - field "foo" is converted to an integer (`mutate { convert { foo => "integer" }}`)

 - elasticsearch requires consistent types, but will attempt to coerce at index time
   - if the first event for an index was A, the index will be created with foo's type as an integer
   - if the first event for an index was B, the index will be created with foo's type as a string.

Additionally, should the whitespace-only string `"    "` be considered empty for this purpose, or only `""`?

Reply:
I agree with you that this has the potential to increase mapping errors.  

I will pull this from this PR and discuss further on #68 , it seems that the reporter really wants a way to differentiate a real zero from an invalid converted one.